### PR TITLE
fix TVA impl (utilize second learning rate)

### PR DESCRIPTION
**Pull request by thnkinbtfly**
Thanks for these cool papers! 👍👍
I was trying to use https://github.com/ethch18/specializing-multilingual , and found `second_learning_rate` is not used.
I think this PR will fix it

### CODE REVIEW

**Comment by ethch18** [comment]
Thanks for this fix!  Do you have data points on how various accuracy measures change once you added the second learning rate?
